webextensions/api/tabs.json

```
@@ -2879,7 +2876,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "28"
```

:x: https://github.com/chromium/chromium/commit/6c75fab15c44f3fdb63be8201b49dc5a40715be1 in [27.0.1442.0](https://github.com/chromium/chromium/releases/tag/27.0.1442.0) before 27.0.1453.

```suggestion
                    "version_added": "27"
```